src/components/service/ServiceCard.tsx

```diff
@@ -22,6 +22,8 @@ const ServiceCard = ({ serviceData, isDiffMode, preState, state }: ServiceCardPr
   const activeService = postService || preService;
 
   const changeType = isDiffMode ? getServiceChangeType(serviceData) : 'normal';
```

Comment:
Seems that `changed` status is not detected correctly (and hasn't been in the past). Can you fix this as well?